cloud/amazon/ec2_customer_gateway.py

```diff
@@ -48,14 +48,21 @@
 '''
 
 EXAMPLES = '''
+
+# Note: You cannot create more than one customer gateway with the same IP address. If you run an identical request more than one time, the first request creates the customer gateway, and subsequent requests return information about the existing customer gateway. The subsequent requests do not create new customer gateway resources.
```

Comment:
Hi,

Applogies, I wasn't clear, you can specify a note in the following form
https://github.com/ansible/ansible-modules-extras/blob/devel/system/debconf.py#L32

So it appears as below the examples, as shown here http://docs.ansible.com/ansible/debconf_module.html